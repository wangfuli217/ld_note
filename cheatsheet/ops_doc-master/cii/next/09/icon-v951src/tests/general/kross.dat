elephants:peanuts
encroachment:roaches
gaggle:geese
