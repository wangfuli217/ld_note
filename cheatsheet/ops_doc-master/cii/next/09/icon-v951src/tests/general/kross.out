

p
elephants
a
n
u
t
s

  p
elephants
  a
  n
  u
  t
  s

elephants
   e
   a
   n
   u
   t
   s

     p
     e
elephants
     n
     u
     t
     s

      p
      e
      a
elephants
      u
      t
      s

       p
       e
       a
       n
       u
elephants
       s

        p
        e
        a
        n
        u
        t
elephants


r
o
a
c
h
encroachment
s

  r
  o
  a
encroachment
  h
  e
  s

encroachment
   o
   a
   c
   h
   e
   s

    r
encroachment
    a
    c
    h
    e
    s

     r
     o
encroachment
     c
     h
     e
     s

      r
      o
      a
encroachment
      h
      e
      s

       r
       o
       a
       c
encroachment
       e
       s

         r
         o
         a
         c
         h
encroachment
         s


gaggle
e
e
s
e

gaggle
  e
  e
  s
  e

gaggle
   e
   e
   s
   e

     g
gaggle
     e
     s
     e

     g
     e
gaggle
     s
     e

     g
     e
     e
     s
gaggle
